kubernetes/crd/domain-crd.yaml

```diff
@@ -29,6 +29,17 @@ spec:
                 type: object
                 description: Models and overrides affecting the WebLogic domain configuration.
                 properties:
+                  overrideDistributionStrategy:
+                    type: string
+                    description: Determines how updated configuration overrides are
+                      distributed to already running WebLogic servers following introspection
+                      when the domainHomeSourceType is PersistentVolume or Image.  Configuration
+                      overrides are generated during introspection from secrets, the
+                      overrideConfigMap field, and WebLogic domain topology.Legal
```

Comment:
Typo missing space after the period -> ". Legal values are DYNAMIC (the default) and ON_RESTART."